### ISSUE TITLE
Motion Sensor slider is not accessible on an iPhone X

### ISSUE THREAD

**Issue by Ross-Gibson** [Triggertrap/triggertrap-ios#18]
**Steps to reproduce**:
- Open the Motion Sensor mode on an iPhone X
- Rotate to portrait
> Notice that the slider is partially blocked by the bottom view
- Rotate to landscape
> Notice that the slider is accessible

**Expected result**:
The slider should to accessible in all rotations, and should sit just above the bottom view.

**Comment by Ross-Gibson** [comment]
Just noticed this is a duplicate of #5
